packages/kbn-server-route-repository/BUILD.bazel

```diff
@@ -52,6 +52,13 @@ jsts_transpiler(
   build_pkg_name = package_name(),
 )
 
+jsts_transpiler(
+  name = "target_web",
+  srcs = SRCS,
```

Comment:
I wonder if a potential improvement could be to only include in this version the `formatRequest` helper to avoid the split imports in the plugins.

Reply:
I went ahead and added only the needed files because `jest` complained that it couldn't reach the dependency `./parse_endpoint` because it was from outside the module (from the import `import { formatRequest } from '@kbn/server-route-repository/target_web/format_request';`)